atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -131,9 +131,9 @@ def allow_inheritance(self):
         self.log.info("Inheritance requested in config file")
 
         if is_scratch_build(self.workflow) or is_isolated_build(self.workflow):
-            self.log.warning(
```

Comment:
I'd like to keep also original log on the server side as a warning

Reply:
I am actually wondering IF logging user warnings shouldn't log also regular warning

Reply:
Good point :+1:
To close it, I opened a [PR](https://github.com/containerbuildsystem/osbs-client/pull/1017) in osbs-client to send warnings automatically.

Reply:
I'm not sure if we want this

Reply:
yeah just create another log as warning as well,

we might want to log different message in user & warn logs

Reply:
Fixed